egs/public_dataset/ofuton_p_utagoe_db/README.md

```diff
@@ -0,0 +1,15 @@
+# ofuton_p_utagoe_db
+
+## Running Instruction
+
+- Train: `sh train.sh` 
```

Comment:
I don't think this is up-to-date. Please refactor it according to the current structure.